src/mono/wasm/runtime/debug.ts

```diff
@@ -124,6 +126,7 @@ export function mono_wasm_raise_debug_event(event: WasmEvent, args = {}): void {
     if (typeof args !== "object")
         throw new Error(`args must be an object, but got ${JSON.stringify(args)}`);
 
+    // eslint-disable-next-line no-console
     console.debug("mono_wasm_debug_event_raised:aef14bca-5519-4dfe-b35a-f867abc123ae", JSON.stringify(event), JSON.stringify(args));
 }
```

Comment:
Are the console calls here left on purpose?

Reply:
they are somehow magical. @thaystg which of them could be removed or unified ?

Reply:
I will merge this and we could improve debug usage of console in the future PRs.